src/com/google/javascript/jscomp/DefaultPassConfig.java

```diff
@@ -2146,7 +2146,12 @@ public void process(Node externs, Node root) {
       PassFactory.builder()
           .setName(PassNames.COLLAPSE_PROPERTIES)
           .setInternalFactory(
-              (compiler) -> new CollapseProperties(compiler, options.getPropertyCollapseLevel()))
+              (compiler) -> new CollapseProperties(
+                  compiler,
+                  options.getPropertyCollapseLevel(),
+                  options.chunkOutputType,
```

Comment:
Nite: Could we add a getter (and setter) for this in CompilerOptions rather than accessing directly?

Reply:
done.